grammars/less.cson

```diff
@@ -300,85 +327,183 @@
     'match': ':'
     'name': 'punctuation.separator.key-value.css'
   }
+  {
+    'match': '\\btrue\\b'
```

Comment:
It doesn't look like it. Basically, only `true` is truthy, everything else is false: http://lesscss.org/features/#mixin-guards-feature-guard-comparison-operators